app/models/world_location_news.rb

```diff
@@ -17,4 +17,20 @@ def title
   def description
     @content_item.content_item_data["description"]
   end
+
+  def ordered_featured_documents
+    @content_item.content_item_data.dig("details", "ordered_featured_documents")&.map do |document|
+      {
+        href: document["href"],
+        image_src: document.dig("image", "url"),
+        image_alt: document.dig("image", "alt_text"),
+        heading_text: document["title"],
+        description: ActionView::Base.full_sanitizer.sanitize(document["summary"]).truncate(160, separator: " "),
```

Comment:
Just out of interest, why are we sanitising here? Partly wondering if we need to do anything similar for e.g. topical events

Reply:
The description text contains HTML in the content item.  If we don't sanitize, then truncate will cut off the closing HTML tags.

Reply:
Ah, do we need the same for topical events [here](https://github.com/alphagov/collections/blob/3aaaf6a662f6b4db60d06fb725af56d906850045/app/models/topical_event.rb#L75) in that case?

Reply:
For some reason, we aren't converting topical events to HTML in Whitehall.  Maybe we should, just to be consistent?

Reply:
Oh yeah - actually I think this consistency will come as part of this [spike ticket](https://trello.com/c/6duIhmSQ/192-spike-using-shared-featurable-code-for-topical-events-instead-of-custom-topicaleventfeaturingscontroller)